fix: Clarify creating source bundles (#6008)
getsentry/sentry-docs: src/docs/product/cli/dif.mdx

## Creating Source Bundles

> To get inline source context in stack traces in the Sentry UI, Sentry needs so-called
> __source bundles__, an archive containing all source files referenced by a specific
> debug information file. You can use sentry-cli to scan debug files for references to
> source code files, resolve them in the local file system and bundle them up in a source
> bundle. Either use the `difutil bundle-sources` command or add the `--include-sources`
> option to the `upload-dif` command.

Reviewer: ```suggestion
To get inline source context in stack traces in the Sentry UI, Sentry needs
__source bundles__, an archive containing all source files referenced by a specific
debug information file. You can use sentry-cli to scan debug files for references to
source code files, resolve them in the local file system, and bundle them up in a source
bundle. 

This is particularly useful when building and uploading debug information files
are detached, because a source bundle can be created when building, and then uploaded with `sentry-cli upload-dif` at any time later.

```

To me, the original way this was written is easier to read. It's confusing to give the instructions "Either use the `difutil bundle-sources` command or add the `--include-sources`
option to the `upload-dif` command." and then show these commands again below. It makes more sense to talk about the commands and then show them immediately afterwards. I also think it makes more sense to mention that this method is particularly useful when building and uploading debug information files are detached, the way we have it in the original.

> ### Dif Util

Reviewer: ```suggestion
### Dif Util

To create a source bundle, first, use the `difutil bundle-sources` command on a list of
debug information files:
```

> This option is handy when building and uploading debug information files are detached.
> 
> First, use the `difutil bundle-sources` command on a list of debug information files to
> create the source bundle. To create multiple source bundles for all debug information
> files, use the command on each file individually.:

Reviewer: ```suggestion

```

```bash
# on the build machine:
sentry-cli difutil bundle-sources /path/to/files...
```

> Then upload the source bundle to Sentry at any later time:

Reviewer: ```suggestion
Then, upload the source bundle to Sentry at any time:
```

```bash
sentry-cli upload-dif --type sourcebundle /path/to/bundles...
```

> ### Include Sources

Reviewer: ```suggestion
To create multiple source bundles for all debug information files, use the
command on each file individually.

### Include Sources
```

> Add the `--include-sources` option to the `upload-dif` command, which generates
> source bundles on the fly during the upload. This requires that the upload is
> performed on the same machine as the application build:

Reviewer: ```suggestion
Alternatively, you can add the `--include-sources` option to the `upload-dif` command, which generates
source bundles on the fly during the upload. This requires that the upload is
performed on the same machine as the application build:
```
